picasso/src/main/java/com/squareup/picasso3/FetchAction.java

```diff
@@ -15,14 +15,13 @@
  */
 package com.squareup.picasso3;
 
-class FetchAction extends Action<Object> {
+import android.support.annotation.NonNull;
 
-  private final Object fetchTarget;
+class FetchAction extends Action {
   private Callback callback;
 
   FetchAction(Picasso picasso, Request data, Callback callback) {
-    super(picasso, null, data);
-    this.fetchTarget = new Object();
+    super(picasso, data);
```

Comment:
any idea why this used to return `new Object()` vs this?  was it to avoid strong references?